ledger/ledger-api-bench-tool/BUILD.bazel

```diff
@@ -75,11 +75,22 @@ da_scala_library(
 da_scala_library(
     name = "ledger-api-bench-tool-test-lib",
     srcs = glob(["src/test/lib/**/*.scala"]),
-    scala_deps = [],
+    scala_deps = [
```

Comment:
If these dependencies were previously coming from `model-tests` why don't we now get them via `benchtool-tests` ?

Reply:
From @pbatko-da - I had to override a method from SanboxFixture for each IT test. So I created a new trait and overrode it there once. That trait lives in ...-lib target and needs  the deps that SandboxFixture needs